drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/window/Dimensions.java

```diff
@@ -0,0 +1,39 @@
+package org.jboss.arquillian.drone.webdriver.window;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Dimensions {
+
+    public static final Pattern DIMENSIONS_PATTERN = Pattern.compile("([0-9]+)x([0-9]+)");
+    private String dimensions;
+    private int width;
+    private int height;
+
+    public Dimensions(String dimensions) {
+        this.dimensions = dimensions;
+    }
+
+    public int getWidth() {
+
+        return width;
+    }
+
+    public int getHeight() {
+        return height;
+    }
+
+    public boolean hasValidDimensionPattern() {
+        Matcher m = DIMENSIONS_PATTERN.matcher(dimensions);
+        if (m.matches()) {
+            setDimensions(m);
```

Comment:
This means that the dimensions variables (`width` and `height`) are set only if the method `hasValidDimensionPattern` is called (this doesn't seem to be very intuitive). I would expect that the variables are tried to be set in the constructor. If it is not valid, then the values are zero. This would lead to the state when in the if statement https://github.com/arquillian/arquillian-extension-drone/pull/98/files#diff-ff926f782beea29de25d18e97da425daR127 you could check if the variables are set or not.